packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1095,6 +1095,1007 @@ curl -v -X DELETE \
 HTTP/1.1 204 No Content
 ```
 
+## Email template operations
+
+The Email Templates API allows you to programatically manage email customizations.
+
+### Email Templates
+
+Okta provides many customizable **email templates**. For example, the `welcome` email template allows users to activate their account. Each template has **default content** that is translated to any one of the [supported languages](#supported-languages).
+
+### Email Customizations
+
+**Email customizations** allow you to override an email template's default content.
+
+The following constraints apply to email customizations:
+- If an email template has any customizations at all, exactly one of them must be the default (where `isDefault` is `true`). The default customization is used when no other customization applies to the user's language settings.
+- Each email template can have only one customization for each [supported language](#supported-languages).
+
+### Supported Languages
+
+Email customizations can be created for the following languages. Language values must be in [BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) format.
```

Comment:
What if we linked to the eng doc source? In the guides and such, we try to avoid linking to Wikipedia.
```suggestion
Email customizations can be created for the following languages. Language values must be in [BCP 47 language tag](http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry) format.
```